### PR TITLE
Code subroutines documentation

### PR DESCRIPTION
**Pull request by samuelowino**
Awaiting comments

### CODE REVIEW

**Review by samuelowino — COMMENTED** [review]
Added JSONObject - JSONArray parser.
Added method blocks documentation.

**Comment by samuelowino** [comment]
Just enough Docs for Generation of Standard Java Doc (doclet) using automated IDE tools

File: api/src/main/java/org/openmrs/module/muzima/handler/DemographicsUpdateQueueDataHandler.java [hunk 1]
```diff
@@ -48,6 +48,11 @@
     private QueueProcessorException queueProcessorException;
     private Patient unsavedPatient;
 
+    /**
+     * 
+     * @param queueData - QueueData
```
Comment (mssavai) [inline api/src/main/java/org/openmrs/module/muzima/handler/DemographicsUpdateQueueDataHandler.java:53]: Needs description of what the method is doing and more description of the quedata param.

File: api/src/main/java/org/openmrs/module/muzima/handler/DemographicsUpdateQueueDataHandler.java [hunk 2]
```diff
@@ -66,6 +71,11 @@ public void process(final QueueData queueData) throws QueueProcessorException {
         }
     }
 
+    /**
+     * 
+     * @param queueData
+     * @return boolean
```
Comment (mssavai) [inline api/src/main/java/org/openmrs/module/muzima/handler/DemographicsUpdateQueueDataHandler.java:77]: Add method description

File: api/src/main/java/org/openmrs/module/muzima/handler/JsonEncounterQueueDataHandler.java [hunk 3]
```diff
@@ -54,6 +54,7 @@
 import java.util.Map;
 
 /**
+ * TODO brief class desceription.
```
Comment (mssavai) [inline api/src/main/java/org/openmrs/module/muzima/handler/JsonEncounterQueueDataHandler.java:57]: Add the class description as well

**Review by mssavai — COMMENTED** [review]
Some public methods are lacking method description, please check first. Thanks

**Pull request merged**